src/UI/templates/default/Card/card.less

```diff
@@ -19,10 +26,26 @@
 	.caption{
 		.card-title{
 			overflow-wrap: break-word;
-		}
 
+		}
 		&:nth-of-type(1n+3) {
 			border-top: 1px solid @panel-inner-border;
 		}
+
 	}
 }
+
+.il-panel-report .il-card{
+	background-color: @il-secondary-container-bg;
+}
+
+.row-eq-height {
+	.il-card {
+		display: flex;
```

Comment:
I like the flexible box layout approach. IE 11 seems to only partly support it (see: https://caniuse.com/#search=flex), but we should force users to use a _state of the art_ browser ;-).